### ISSUE TITLE
Button doesn't seem to get updates when there's a textfield in alert content

### ISSUE THREAD

**Issue by ynie** [divadretlaw/CustomAlert#15]
Hi, thank you for the great framework. I run into some issues where the alert buttons didn't get updated. Here's the code:

```
      .customAlert("Enter access code",
                     isPresented: $isShowingAccessCodeAlert,
                     content: {
            TextField("Access Code", text: $accessCode)
                .onSubmit {
                    submitCode()
                    isShowingAccessCodeAlert = false
                }
        }, actions: {
            MultiButton {
                Button(role: .cancel) {
                } label: {
                    Text("Cancel")
                }
                
                Button("Submit", action: submitCode)
                    .disabled(accessCode.trimmingCharacters(in: .whitespacesAndNewlines).count == 0)
                    .bold()
            }
        })
```

it doesn't seem like the button disable state is getting updated when the user starting to enter text. What did I do wrong? Thanks!

**Comment by divadretlaw** [comment]
Hi, well the _current_ version doesn't support this due to a limitation of `WindowKit` which is used to display the alert.
I found a way around this limitation in `WindowKit` 2.1.0+ but for reasons unknown to me at the moment, the button don't update, even though they get redrawn.

I will continue to investigate this issue, hopefully I can find a solution, because this behavior seems very useful.

**Comment by ynie** [comment]
I fixed the issue by introducing a env variable and force the button to update. It's not great, but it does the trick case by case.

**Comment by divadretlaw** [comment]
Hi, this should be fixed in `3.5.0`

![Simulator Screen Recording - iPhone 15 - 2024-03-26 at 22 20 16](https://github.com/divadretlaw/CustomAlert/assets/6899256/4d247202-dfa9-49de-bed7-4640e95ec768)

**Comment by ynie** [comment]
Thank you
